website/docs/basics/blocks-wrapper.md

```diff
@@ -5,3 +5,91 @@ sidebar_label: Wrapper
+
+### wrapperBackgroundColor
+
+Controls the background color of your sections. You can change it by providing the colors registered in the block editor's color palette.
+
+### wrapperWidth (Responsive)
+
+Defines the width of the content (block) inside of the wrapper. It behaves by a grid of 12 (default), and it starts from the left edge.
+
+### wrapperOffset (Responsive)
+
+The same as the width, but this attribute controls the offset (margin) from the left of your wrapper.
+
+### wrapperContainerWidth (Responsive)
+
+This attribute defines your block's container width, and on that container, wrapper width and wrapperContainerWidth be applied. You can define these values in the wrappers manifest and the global manifest settings.
+
+### wrapperGutter (Responsive)
+
+Controls the left/right spacing for the content (block). You can define these values in the wrappers manifest and the global manifest settings.
+
+### wrapperSpacingTop (Responsive)
+
+Moves the spacing from the top by an increment of 10px. It uses margin-top to set a spacing, background color **will not be applied to this attribute**. You can define these values in the wrappers manifest and the global manifest settings.
+
+### wrapperSpacingBottom (Responsive)
+
+Moves the spacing from the bottom by an increment of 10px. It uses margin-bottom to set a spacing, background color **will not be applied to this attribute**. You can define these values in the wrappers manifest and the global manifest settings.
+
+### wrapperSpacingTopIn (Responsive)
+
+Moves the spacing from the top by an increment of 10px. It uses padding-top to set a spacing, background color **will be applied to this attribute**. You can define these values in the wrappers manifest and the global manifest settings.
+
+### wrapperSpacingBottomIn (Responsive)
+
+Moves the spacing from the bottom by an increment of 10px. It uses padding-bottom to set a spacing, background color **will be applied to this attribute**. You can define these values in the wrappers manifest and the global manifest settings.
+
+### wrapperDividerTop (Responsive)
+
+This attribute adds a divider on the top of the section. The spacing of the divider from the content is set by `wrapperSpacingTopIn` attributes.
+
+### wrapperDividerBottom (Responsive)
+
+This attribute adds a divider on the bottom of the section. The spacing of the divider from the content is set by `wrapperSpacingBottomIn` attributes.
+
+### wrapperHide (Responsive)
+
+Controls will the section be showed in a specific breakpoint.
+
+> Important note that all responsive options are made in `desktop` first way for the block editor because every editor wants to style these firsts. All of the responsive attributes and applied from the large to the mobile breakpoint. So if you set the tablet's value, it will be applied to the tabled and mobile breakpoint.
```

Comment:
```suggestion
> **Important note**: all responsive options are made in `desktop` first way for the block editor because every editor wants to style these firsts. All of the responsive attributes are applied from the large to the mobile breakpoint. So if you set the tablet values, it will be applied to the tablet and mobile breakpoints.
```